operator_service/utils.py

```diff
@@ -55,13 +58,45 @@ def check_required_attributes(required_attributes, data, method):
     return None, None
 
 
-def process_signature_validation(signature, original_msg):
+def get_signer(signature, message):
+    """
+    :return: True if signature is valid
+    """
+    # db_nonce = get_nonce(signer_address)
+    # if db_nonce and float(nonce) < float(db_nonce):
+    #    msg = (
+    #        f"Invalid signature expected nonce ({db_nonce}) > current nonce ({nonce})."
+    #    )
+    #    logger.error(msg)
+    #    raise InvalidSignatureError(msg)
+
+    # address = Account.recover_message(encode_defunct(text=message), signature=signature)
+    signature_bytes = Web3.toBytes(hexstr=signature)
+    if signature_bytes[64] == 27:
+        new_signature = b"".join([signature_bytes[0:64], b"\x00"])
+    elif signature_bytes[64] == 28:
+        new_signature = b"".join([signature_bytes[0:64], b"\x01"])
+    else:
+        new_signature = signature_bytes
+    signature = keys.Signature(signature_bytes=new_signature)
+    message_hash = Web3.solidityKeccak(
+        ["bytes"],
+        [Web3.toHex(Web3.toBytes(text=message))],
```

Comment:
You don't need both `toHex` and `toBytes` here. Please simplify to just use one or the other. I recommend using `Web3.toBytes(text=message)` for consistency line 88 below

```python
>>> from web3.main import Web3
>>> msg = "hello"
>>> Web3.solidityKeccak(["bytes"], [Web3.toHex(Web3.toBytes(text=msg))])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
>>> Web3.solidityKeccak(["bytes"], [Web3.toBytes(text=msg)])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
>>> Web3.solidityKeccak(["bytes"], [Web3.toHex(text=msg)])
HexBytes('0x1c8aff950685c2ed4bc3174f3472287b56d9517b9c948127319a09a7a36deac8')
```